git_machete/cli.py

```diff
@@ -732,9 +755,11 @@ def strip_remote_name(remote_branch: RemoteBranchShortName) -> LocalBranchShortN
             machete_client.slide_out(
                 branches_to_slide_out=list(map(LocalBranchShortName.of, branches)),
                 opt_down_fork_point=cli_opts.opt_down_fork_point,
+                opt_yes=cli_opts.opt_yes,
                 opt_merge=cli_opts.opt_merge,
                 opt_no_interactive_rebase=cli_opts.opt_no_interactive_rebase,
-                opt_no_edit_merge=cli_opts.opt_no_edit_merge)
+                opt_no_edit_merge=cli_opts.opt_no_edit_merge,
+                delete_git_branches=parsed_cli.delete if 'delete' in parsed_cli else False)
```

Comment:
Pls add that to `cli_opts` as `opt_delete` for the sake of Sacred Consistency ™️